### DISCUSSION TITLE
Daily 2/11

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [rpgrca/proyecto-ispc-fullstack#267] [General]
Ayer estuve realizando el footer para colocarlo en los html, esto me llevó mas tiempo del que esperaba.
Hoy voy a estar subiendolo al repositorio y copiandolo en todos los archivos. Mas tarde voy a estar armando el speach para el video que armaremos el domingo

**Comment by rpgrca** [comment]
Ayer terminé los trámites para conseguir el sitio wordpress, hoy probablemente va los queries y empiece a chusmear lo de Wordpress. Y ponerme a actualizar los diagramas.

**Comment by AhumadaCeleste** [comment]
Últimamente estoy con poca disponibilidad de tiempo, lo cual recupero con mucha intensidad los fines de semana. Después de renegar bastante logre hacer la actividad de wordpress y enviar los print al foro. Debo agregar algunas entradas más en  "Tecnologías" Para dejarlo listo para publicar. Me decepciono el resultado un poco, dado a que instale varios temas que me agradaban visualmente, pero no cumplían con la consigna de "poseer un subtitulo" y no pude agregarlo.  Probé diferentes temas, se dañaba algo y debía volver a construir por incompatibilidad entre ellos. Me siento atrasada con lo nuevo de SGE publicado y con Java dado a que estoy priorizando requisitos y debo iniciar el modulo final de Python. Como ya había grabado un video pero muy extenso considero tengo algo mas o menos listo para ello. Mañana quisiera poder hacer alguna funcionalidad de Java para agregar y continuar con Cisco lo que inicio ahora.

**Reply by rpgrca** [reply]
Javascript ;-)

**Comment by Brian-Almada** [comment]
Ayer no dispuse de mucho tiempo. Hoy estuve con el porfolio personal en Wordpress.
